src/System.Collections.Immutable/tests/ImmutableSortedDictionaryTest.cs

```diff
@@ -450,8 +450,8 @@ public void EnumerationPerformance()
             Debug.WriteLine("Timing:{0}{1}", Environment.NewLine, timingText);
         }
 
-        ////[Fact] // not really a functional test -- but very useful to enable when collecting perf traces.
-        public void EnumerationPerformance_Empty()
+        // [Fact] // not really a functional test -- but very useful to enable when collecting perf traces.
```

Comment:
Then we should make these ConditionalFacts on, for example, some environment variable being set.